test/e2e/helm/kuma_helm_upgrade_multizone.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 func UpgradingWithHelmChartMultizone() {
+	namespace := "helm-upgrade-ns"
```

Comment:
shouldn't we randomize this? so that we don't have to wait for ns deletion?

Reply:
The k8s cluster is the same so we do need to wait for ns deletion anyway